var/spack/repos/builtin/packages/mapl/package.py

```diff
@@ -349,6 +349,8 @@ def cmake_args(self):
 
         if self.spec.satisfies("^mpich"):
             args.append(self.define("MPI_STACK", "mpich"))
+        elif self.spec.satisfies("^mvapich2"):
+            args.append(self.define("MPI_STACK", "mvapich"))
```

Comment:
Just to confirm, the value doesn't need to be `mvapich2`, correct?

Reply:
I think that is correct, yes. @mathomp4 please confirm!

Reply:
Yes. Internally it wouldn't care about `mvapich2` vs `mvapich`. It mainly sets things like the right mpirun command, environment variables, etc.